src/sider/aktivitetskrav/historikk/AktivitetskravHistorikk.tsx

```diff
@@ -29,7 +29,8 @@ const isRelevantForHistorikk = (vurdering: AktivitetskravVurderingDTO) =>
   vurdering.status === AktivitetskravStatus.STANS ||
   vurdering.status === AktivitetskravStatus.IKKE_OPPFYLT ||
   vurdering.status === AktivitetskravStatus.FORHANDSVARSEL ||
-  vurdering.status === AktivitetskravStatus.AVVENT;
+  vurdering.status === AktivitetskravStatus.AVVENT ||
+  vurdering.status === AktivitetskravStatus.IKKE_AKTUELL;
```

Comment:
Tror det er en funksjon under her som må tilpasses for å bygge riktig historikk-tekst for `IKKE_AKTUELL`